rollup/sync_service.go

```diff
@@ -1007,10 +1007,6 @@ func (s *SyncService) ProcessSequencerBatchAppendedLog(ctx context.Context, ethl
 	for i, element := range cd.ChainElements {
 		var tx *types.Transaction
 		index := (event.TotalElements.Uint64() - uint64(len(cd.ChainElements))) + uint64(i)
```

Comment:
Bugfix: keep additional counter, if not sequenced, increment, add that counter to starting queue index to calculate queue index for cache